### PR TITLE
fix(docs): update const error-code in syscall page

### PR DESCRIPTION
**Pull request by RetricSu**

### CODE REVIEW

File: website/docs/script/syscalls_for_script.mdx [hunk 1]
```diff
@@ -48,13 +48,18 @@ A collection of CKB-VM syscalls. Also include relevant constant, such as return
...
-| --------- | ---------------------- | ---------------------------------------------- |
-| 0         | CKB_SUCCESS            | No error.                                      |
-| 1         | CKB_INDEX_OUT_OF_BOUND | Index out of bound. (e.g. No such input Cell.) |
-| 2         | CKB_ITEM_MISSING       | The requested resource does not exist.         |
-| 3         | CKB_LENGTH_NOT_ENOUGH  | The supplied memory buffer too small.          |
-| 4         | CKB_INVALID_DATA       | The data provided is invalid.                  |
+| Const No. | C Example              | Description                                              |
+| --------- | ---------------------- | -------------------------------------------------------- |
+| 0         | CKB_SUCCESS            | No error.                                                |
+| 1         | CKB_INDEX_OUT_OF_BOUND | Index out of bound. (e.g. No such input Cell.)           |
```
Comment (linnnsss) [inline website/docs/script/syscalls_for_script.mdx:54]: Index out of bounds (e.g., no such input Cell).

File: website/docs/script/syscalls_for_script.mdx [hunk 2]
```diff
@@ -48,13 +48,18 @@ A collection of CKB-VM syscalls. Also include relevant constant, such as return
...
+| 2         | CKB_ITEM_MISSING       | The requested resource does not exist.                   |
+| 3         | CKB_LENGTH_NOT_ENOUGH  | The supplied memory buffer too small.                    |
```
Comment (linnnsss) [inline website/docs/script/syscalls_for_script.mdx:56]: The supplied memory buffer is too small.

**Review by yfeng2824 — APPROVED** [review]

File: website/docs/script/syscalls_for_script.mdx (hunk 1 above)
Comment (RetricSu) [reply website/docs/script/syscalls_for_script.mdx]: updated

File: website/docs/script/syscalls_for_script.mdx (hunk 2 above)
Comment (RetricSu) [reply website/docs/script/syscalls_for_script.mdx]: updated

**Pull request merged**